v3.0/getting-started/kubernetes/upgrade/convert.md

```diff
@@ -3,5 +3,55 @@ title: Converting your calicoctl manifests
 redirect_from: latest/getting-started/kubernetes/upgrade/convert
```

Comment:
To pass CI, you could add `no_canonical: true` here.

Reply:
ah I thought I added it here, lemme do that

Reply:
hmm, not sure why the CI is still failing. The link it's complaining about has no_canonical thingy already. Anyway I've squashed

Reply:
Sorry....one more small thing. I think if you remove the `redirect_from` it will pass.

Reply:
I have not had the two tags together before, I guess. The issue is that `redirect_from` causes Jekyll to create an actual HTML pages that performs the redirect. So, that is why it is failing.

Reply:
oh, but how will it redirect if I remove the `reditect_from`?

Reply:
Well it is a good point....I think we would need to add the `redirect_from` afterwards (and also remove `no_canonical`, because chicken egg problem. 😢  hoping to clean this up a little bit very soon!

Reply:
We basically need a `no_canonical` in the latest file, but there is no MD source for it. So, we can just remove the `redirect_from` and then it is fine.